Extension/src/Automation/Async/manual-signal.ts

```diff
@@ -0,0 +1,94 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { ManualPromise } from './manual-promise';
+import { Resetable } from './resolvable';
+
+/**
+ * A signal is an externally fulfillable promise ( @see ManualPromise )
+ * that once awaited, will automatically reset to the unwaited state
```

Comment:
I think this is a copy/paste error from `signal.ts`

Reply:
Fixing...